### PR TITLE
Add option for less verbosity

### LINKED ISSUE #2: Less Verbose
The plugin is awesome! Thank you very much for it! But could you please add an option to run in silent mode!?

### PR DESCRIPTION
**Pull request by gerardroche**
It's not too pretty but it works.

Closes #2

### CODE REVIEW

**Comment by randy3k** [comment]
Thanks, I don't have time to review it now. May be tomorrow.

File: package_reloader.py [hunk 1]
```diff
@@ -60,6 +62,12 @@ def run_async(self, pkg_name=None):
         pkg_name = self.current_package_name
 
         if pkg_name:
+
+            # Clear any package loggers
+            logger = logging.getLogger(pkg_name)
+            for handler in list(logger.handlers):
+                logger.removeHandler(handler)
```
Comment (randy3k) [inline package_reloader.py]: I am not sure if I like this idea. It should be handled by the package but not APR.

**Comment by randy3k** [comment]
I just realize that my "tomorrow" is 18 days later...

File: package_reloader.py (hunk 1 above)
Comment (gerardroche) [reply package_reloader.py]: Sorry, I didn't mean this to part of the last PR.

**Comment by gerardroche** [comment]
I've reset the PR to exclude the logger related code.

**Comment by randy3k** [comment]
sorry for the long delay.

**Pull request merged**